scipy/spatial/transform/_rotation.pyx

```diff
@@ -289,77 +344,92 @@ cdef double[:, :] _compute_euler_from_quat(
         
     # Step 0
     # Check if permutation is even (+1) or odd (-1)     
-    cdef int sign = (i - j) * (j - k) * (k - i) // 2
+    cdef int sig = (i - j) * (j - k) * (k - i) // 2
```

Comment:
Why did you rename it?

Reply:
No particular reason, just felt it looked tidier and made some lines fit better. 
Do you prefer how it was before?

Reply:
Well, It might be confusing for someone not familiar with the code and the algorithm. I don't know if "sig" is a common shorthand for "sign".

Reply:
I see, I was actually the one that introduced this in the other PR, but maybe sign is indeed better, will return it to how it was